app/controllers/players_controller.rb

```diff
@@ -33,4 +33,13 @@ def check_if_game_is_full
 
     redirect_to root_path, alert: "Sorry, the game ‘#{@game.slug}’ is full."
   end
+
+  def no_arthur?
```

Comment:
Can you put this logic into the model, to help us keep logic out of the controller?

Reply:
'thin controllers' is a RoR catchphrase

Reply:
I had never head that before, I assumed generic code like this went into the controller, I'll move it into the model